sdk/core/azure-json/README.md

```diff
@@ -52,137 +52,81 @@ could be called indefinitely returning the same integer without error until `nex
-        return jsonReader.readObject(reader -> {
-            // Since this class has no constructor reading to fields can be done inline.
-            // If the class had a constructor with arguments the recommendation is using local variables to track
-            // all field values.
-            JsonSerializableExample result = new JsonSerializableExample();
-            while (reader.nextToken() != JsonToken.END_OBJECT) {
-                String fieldName = reader.getFieldName();
-                if ("anInt".equals(fieldName)) {
-                    result.anInt = reader.getInt();
-                } else if ("aBoolean".equals(fieldName)) {
-                    result.aBoolean = reader.getBoolean();
-                } else if ("aString".equals(fieldName)) {
-                    result.aString = reader.getString();
-                } else if ("aNullableDecimal".equals(fieldName)) {
-                    // getNullable returns null if the current token is JsonToken.NULL, if the current token isn't
-                    // JsonToken.NULL it passes the reader to the ReadValueCallback.
-                    result.aNullableDecimal = reader.getNullable(JsonReader::getDouble);
-                } else {
-                    // Skip children when the field is unknown.
-                    // If the current token isn't an array or object this is a no-op, otherwise is skips the entire
-                    // sub-array/sub-object.
-                    reader.skipChildren();
-                }
-            }
-
-            return result;
-        });
-    }
-}
-```
+`JsonReader` and `JsonWriter` are service provider interfaces used by `JsonProvider` and `JsonProviders` to enable
+implementations to be loaded from the class path. The Azure JSON package provides a default implementation that is
+used if one can't be found on the class path. To provide a custom implementation, implement `JsonReader`, `JsonWriter`,
+and `JsonProvider` in your own package and indicate that the package provides an instance of `JsonProvider`. To ensure
+that your implementations are correct, include the `test` scoped dependency of Azure JSON and extend the
+`JsonReaderContractTests`, `JsonWriterContractTests`, and `JsonProviderContractTests`. These tests outline all basic
+contract requirements set forth by `JsonReader`, `JsonWriter`, and `JsonProvider` and testing a few complex scenarios 
+to provide validation of any implementation.
 
-## Next steps
+## Examples
 
-Get started with Azure libraries that are [built using Azure Core](https://azure.github.io/azure-sdk/releases/latest/#java).
+Check out the [samples README][samples_readme] for in-depth examples on how to use Azure JSON.
 
 ## Troubleshooting
 
 If you encounter any bugs, please file issues via [GitHub Issues](https://github.com/Azure/azure-sdk-for-java/issues/new/choose)
 or checkout [StackOverflow for Azure Java SDK](https://stackoverflow.com/questions/tagged/azure-java-sdk).
```

Comment:
Are we now pointing to StackOverflow in all our troubleshooting guides?

Reply:
This was just copied from existing READMEs, so I guess so?